recipe/symfony4.php

```diff
@@ -10,7 +10,7 @@
 require_once __DIR__ . '/common.php';
 
 set('shared_dirs', ['var/log', 'var/sessions']);
-set('shared_files', ['.env']);
+set('shared_files', ['.env.local']);
```

Comment:
Will it break existing Symfony installations which are using this recipe?

Reply:
In facts, .env is versionned since Symfony 4.1.

Your configuration is ONLY for symfony 4.0 (https://symfony.com/doc/4.1/configuration/dot-env-changes.html) , and this version is not maintained anymore (https://symfony.com/roadmap)

So... yes it will maybe break existing symfony 4.0 apps, but symfony 4.0 isn't maintained so... I think you should adapt your config on the maintained version of symfony

As you want :)